.github/stale.yml

```diff
@@ -0,0 +1,12 @@
+daysUntilStale: 60
+daysUntilClose: 7
+exemptLabels:
+  - pinned
+  - security
+staleLabel: wontfix
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
```

Comment:
is there a way for an issue reporter to specify that their issue should not be closed because it is not "stale"? Are they expected to just keep commenting? I see that there are `exemptLabels` but I am not sure if an issue reporter can set labels at all.

Reply:
Yes, `exemptLabels` does this. The idea is that only maintainers can exempt issues or PRs. If we let users do that, it defeats the purpose.